x/ibc/24-host/validate.go

```diff
@@ -26,8 +36,12 @@ func defaultIdentifierValidator(id string, min, max int) error { //nolint:unpara
 		return sdkerrors.Wrapf(ErrInvalidID, "identifier %s has invalid length: %d, must be between %d-%d characters", id, len(id), min, max)
 	}
 	// valid id must contain only lower alphabetic characters
-	if !sdk.IsAlphaLower(id) {
-		return sdkerrors.Wrapf(ErrInvalidID, "identifier %s must contain only lowercase alphabetic characters", id)
+	if !IsValidID(id) {
+		return sdkerrors.Wrapf(
+			ErrInvalidID,
+			"identifier %s must contain only alphanumeric or the following characters: '.', '_', '+', '-', '#', '[', ']', '<', '>'",
```

Comment:
I'm guessing I need to update the relayer random ID generator to include these characters? Where is the spec link?

Reply:
https://github.com/cosmos/ics/pull/416/files

Reply:
you can use the regex that I defined for the relayer 👍